README.md

```diff
@@ -541,7 +541,7 @@ chflags nohidden ~/Library
 
 #### Increase Number of Recent Places
 ```bash
-defaults write .GlobalPreferences NSNavRecentPlacesLimit -int 10 && \
```

Comment:
I do not have a reference. Typing `defaults read .GlobalPreferences` returns the same preferences as `-g`, `-globalDomain` and `NSGlobalDomain`. My guess is that `defaults` looks for the name of the file in `~/Library/Preferences`. The global preferences are stored in `~/Library/Preferences/.GlobalPreferences.plist`.